packages/cozy-mespapiers-lib/src/components/Placeholders/FeaturedPlaceholdersList.jsx

```diff
@@ -27,11 +27,11 @@ const FeaturedPlaceholdersList = ({ featuredPlaceholders }) => {
   const redirectPaperCreation = placeholder => {
     hideImportDropdown()
     const countrySearchParam = `${
-      placeholder.country ? `country=${placeholder.country}&` : ''
+      placeholder.country ? `country=${placeholder.country}` : ''
     }`
     return navigate({
-      pathname: `/paper/create/${placeholder.label}`,
-      search: `${countrySearchParam}backgroundPath=${location.pathname}`
+      pathname: `${pathname}/create/${placeholder.label}`,
```

Comment:
Is it possible to use a relative path such as `create/${placeholder.label}` here (and possibly other places)? Or do we prefer being explicit and always provide absolute paths?

Reply:
En effet, bien vu 👍